src/test/java/org/fundacionjala/sfdc/hooks/Hook.java

```diff
@@ -0,0 +1,12 @@
+package org.fundacionjala.sfdc.hooks;
+
+import cucumber.api.java.After;
+import org.fundacionjala.sfdc.commons.DriverManager;
+
+public class Hook {
+
+    @After
+    public void finishScenario() {
+        DriverManager.getInstance().quitWebDriver();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)